src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java

```diff
@@ -109,6 +112,7 @@ protected long getTimestampSeconds(String regexPattern) throws ParsingException
     public abstract List<AudioStream> getAudioStreams() throws IOException, ExtractionException;
     public abstract List<VideoStream> getVideoStreams() throws IOException, ExtractionException;
     public abstract List<VideoStream> getVideoOnlyStreams() throws IOException, ExtractionException;
+    public abstract Subtitles[] getSubtitles() throws IOException, ExtractionException, JsonParserException;
```

Comment:
Could you please use a list instead of an array like we're already doing everywhere?

Reply:
Sure, no problem.